### DISCUSSION TITLE
Build from source

### DISCUSSION THREAD

**Discussion by picmauro** [garethgeorge/backrest#408] [Q&A] [answered]
Hi, can you explain how build from source the latest version of Backrest.
I've already installed go,
tnx

**Comment by garethgeorge** [comment, marked as answer]
Hey, build from source is covered by https://github.com/garethgeorge/backrest/blob/main/CONTRIBUTING.md

**Reply by picmauro** [reply]
Thanks, Building ok for my Raspberry Pi 4.
